doc/parameters.md

```diff
@@ -84,7 +84,8 @@ Allows to set the log level for the logs related to OpenID Connect integration
 **Values:**
 
 - `disabled`: completely disabled, just listens on the port
-- `status`: only the `/status/` endpoints enabled for health checks
+- `status`: enables the `/status/` endpoints enabled for health checks and `/policies` that shows the list of available policies.
```

Comment:
```suggestion
- `status`: enables the `/status/` endpoints for health checks and `/policies` that show the list of available policies.
```

Reply:
:man_facepalming: 
Thanks!

Reply:
Sorry, I didn't notice the second change. It should be "shows" instead of "show" right?

Reply:
Just to clarify: which one(s) show the list of available policies?
a. `/policies`
b. health check and `/policies`

In both cases, the verb is `show`, because of plural. However, the sentence might need to be rephrased to avoid ambiguity.

Reply:
It's `a)`, but I think it should be singular because it's the `/policies` endpoint.

Reply:
Suggestion:
status`: enables the `/status/` endpoints for health checks, and the `/policies` endpoint that shows the list of available policies.

Reply:
:+1: